src/components/DraftOrdersTab.tsx

```diff
@@ -22,12 +22,17 @@ import { DraftOrder } from "#/lib/types";
 import { OrderDropdownMenuCell } from "./OrderDropdownMenuCell";
 import { RemoveDraftOrdersDialog } from "./RemoveDraftOrdersDialog";
 import { ReviewOrdersDialog } from "./ReviewOrdersDialog";
+import { StatusBadge } from "./StatusBadge";
 
 export function DraftOrdersTab() {
   const draftOrders = useDraftOrders((state) => state.draftOrders);
   const setReviewDialogOpen = useUIStore((state) => state.setReviewDialogOpen);
   const [selectedIds, setSelectedIds] = useState<string[]>([]);
 
+  const selectableOrders = draftOrders.filter(
+    (order) => order.status == "draft",
```

Comment:
when would their status not be `draft` ? 🤔

Reply:
When it is `creating`